server/server_signals.go

```diff
@@ -0,0 +1,38 @@
+// +build !windows
+
+package server
+
+import (
+	"os/signal"
+	"syscall"
+
+	"github.com/containous/traefik/log"
+)
+
+func (server *Server) configureSignals() {
+	signal.Notify(server.signals, syscall.SIGINT, syscall.SIGTERM, syscall.SIGUSR1)
```

Comment:
`SIGUSR1` isn't present in the Windows build of Go, so the code won't compile.  That was the reason for having `server_signals_windows.go` in the first place.